pkg/cmd/pulumi/package_add.go

```diff
@@ -291,18 +352,22 @@ func csharpPackageName(pkgName string) string {
 
 // Prints instructions for linking a locally generated SDK to an existing .NET
 // project, in the absence of us attempting to perform this linking automatically.
-func printDotnetLinkInstructions(root string, pkg *schema.Package, out string) error {
+func linkDotnetPackage(root string, pkg *schema.Package, out string) error {
 	fmt.Printf("Successfully generated a .NET SDK for the %s package at %s\n", pkg.Name, out)
 	fmt.Println()
-	fmt.Println("To use this SDK in your .NET project, run the following command:")
-	fmt.Println()
+
 	relOut, err := filepath.Rel(root, out)
 	if err != nil {
 		return err
 	}
 
-	fmt.Printf("  dotnet add reference %s\n", filepath.Join(".", relOut))
-	fmt.Println()
+	cmd := exec.Command("dotnet", "add", "reference", relOut)
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("dotnet error: %w", err)
+	}
+
 	fmt.Printf("You also need to add the following to your .csproj file of the program:\n")
 	fmt.Println()
 	fmt.Println("  <DefaultItemExcludes>$(DefaultItemExcludes);sdks/**/*.cs</DefaultItemExcludes>")
```

Comment:
Can't we do this part automatically as well?

Reply:
I can try, I was short on time and needed to get back to main work for now, Mind if i just file a followup issue?

Reply:
Given the dotnet stuff isn't tested yet just back out the changes for this, we can leave it as a "just print" for the time being like java.

Reply:
This comment still stands, I don't think we should try changing half of dotnet, especially without tests. Just leave it as is for now.